setup.js

```diff
@@ -0,0 +1,88 @@
+const axios = require('axios');
+const spaceImport = require('contentful-import');
+const { writeFileSync } = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+
+const argv = require('yargs-parser')(process.argv.slice(2));
+
+const spaceId = argv.spaceId;
+const managementToken = argv.managementToken;
+const deliveryToken = argv.deliveryToken;
+
+// we need all 3 variables in order to set up user's space correctly
+// spaceId and deliveryToken (CDA token) – to write into config file,
+// and managementToken for contentful-import
+if (!spaceId || !managementToken || !deliveryToken) {
+  console.log('');
+  console.log(`You have to provide ${chalk.yellow('spaceId')}, ${chalk.yellow('managementToken')} and ${chalk.yellow('deliveryToken')} arguments in order to set up your space correctly`);
+  console.log('It should look like this: npm run setup -- --spaceId YOUR_SPACE --deliveryToken YOUR_CDA_KEY --managementToken YOUR_CMA_KEY ');
+  console.log('');
+  process.exit('');
```

Comment:
Please exit with an error code. `1` should suffice.

Reply:
exactly, error code above zero should be sent

Reply:
Thanks for noticing, good catch!